astropy/coordinates/tests/test_arrays.py

```diff
@@ -41,10 +42,16 @@ def test_angle_arrays():
     npt.assert_almost_equal(a6.value, 945.0)
     assert a6.unit is u.degree
 
-    with pytest.raises(TypeError):
+    with ExitStack() as stack:
+        stack.enter_context(pytest.raises(TypeError))
         # Arrays where the elements are Angle objects are not supported -- it's
         # really tricky to do correctly, if at all, due to the possibility of
         # nesting.
+        if not NUMPY_LT_1_18:
```

Comment:
this may need to be `NUMPY_LT_1_19` now, but since everything passed, including the np 1.18 job, maybe it's still OK like this?